kits/sql-query-explainer/prompts/sql-query-explainer_explain-query_user.md

```
@@ -0,0 +1,3 @@
+SQL Query:
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)